app/models/web_hook_events_daily_aggregate.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class WebHookEventsDailyAggregate < ActiveRecord::Base
+  belongs_to :web_hook
+
+  default_scope { order("created_at DESC") }
+  after_create :aggregate
+
+  def self.purge_old
```

Comment:
This site setting is for the web hook events themselves. We would need a new `retain_web_hook_events_aggregate_days` setting that defaults to 720 days as per the spec on Meta.

Reply:
We should also make a new job like https://github.com/discourse/discourse/blob/48c58499f1399baf88b43762d39bb0849eb00cf0/app/jobs/scheduled/purge_old_web_hook_events.rb for this